README.md

```diff
@@ -0,0 +1,176 @@
+# ROCM-CORE Introduction
+
+rocm-core is a utility which can be used to get ROCm release version. 
+It also provides the Lmod modules files for the ROCm release.
```

Comment:
It provides an **example** module file.

Reply:
Updated text description to address the comments in the latest version